ideascube/models.py

```diff
@@ -311,6 +311,13 @@ def get_prep_value(self, value):
         return json.dumps(value)
 
 
+class LanguageField(models.CharField):
+    def __init__(self, *args, **kwargs):
+        kwargs['choices'] = get_all_languages()
+
```

Comment:
I strongly disagree with it.

Compare:

```python
if foo:
    do_something()
elif bar:
    do_something_else()
else:
    do_something_different()
```

With:

```python
if foo:
    do_something()

elif bar:
    do_something_else()

else:
    do_something_different()
```

Both are correct according to pep8. Yet the former is much less readable.

Blocks are like paragraphs in prose. Clearly delineating them is important to readability, otherwise the reader needs to delineate them mentally and that requires more mental effort.

Python is all about that: readability, and reducing the required mental effort to understand a piece of code. Therefore, I contend that proper use of whitespace, even when not required, is in fact the Pythonic thing to do.

Now of course, this is an argument against « we shouldn't add blank lines if not requested by pep8 » and is entirely unrelated to the specific case at hand in this review, but you moved the discussion away from it. :wink: 

---

As for this specific line, I don't care one bit, as I thought was clear from my original response, as it didn't contain any strong argument in favour of it and was written in an introspective, unassertive fashion.